src/mon/Elector.cc

```diff
@@ -37,7 +37,11 @@ void Elector::init()
   epoch = mon->store->get(Monitor::MONITOR_NAME, "election_epoch");
   if (!epoch)
     epoch = 1;
-  dout(1) << "init, last seen epoch " << epoch << dendl;
+  dout(1) << "init, last seen epoch " << epoch << ", bumping" << dendl;
+  ++epoch;
```

Comment:
I forget the rules but we're careful about odd and even epochs and this appears to be violating that, since it bumps us from 1 to 2 on first boot.